### ISSUE TITLE
↔ [Converter] Add operations to accelerate Transformer encoder

### ISSUE THREAD

**Issue by narendasan** [pytorch/TensorRT#1754]
## torch.nn.TransformerEncoder

- **Function Schema**:

  - aten::chunk(Tensor(a -> *) self, int chunks, int dim=0) -> Tensor(a)[]
  - aten::_native_multi_head_attention(Tensor query, Tensor key, Tensor value, int embed_dim, int num_head, Tensor qkv_weight, Tensor qkv_bias, Tensor proj_weight, Tensor proj_bias, Tensor? mask=None, bool need_weights=True, bool average_attn_weights=True, int? mask_type=None) -> (Tensor, Tensor)
  - aten::add.t(t[] a, t[] b) -> t[]
  - aten::all.bool(bool[] self) -> bool
  - prim::requires_grad(Tensor a) -> bool
  - aten::any.bool(bool[] self) -> bool
  - aten::_transformer_encoder_layer_fwd(Tensor src, int embed_dim, int num_heads, Tensor qkv_weight, Tensor qkv_bias, Tensor proj_weight, Tensor proj_bias, bool use_gelu, bool norm_first, float eps, Tensor norm_weight_1, Tensor norm_bias_1, Tensor norm_weight_2, Tensor norm_bias_2, Tensor ffn_weight_1, Tensor ffn_bias_1, Tensor ffn_weight_2, Tensor ffn_bias_2, Tensor? mask=None, int? mask_type=None) -> Tensor
  - aten::len.str(str s) -> int
  - aten::str(t elem) -> str
  - prim::device(Tensor a) -> Device
  - aten::is_grad_enabled() -> bool
  - aten::find(str self, str substr, int start=0, int end=-1) -> int
  - prim::is_cuda(Tensor a) -> bool

- **Original PyTorch API**:

  - https://pytorch.org/docs/stable/generated/torch.nn.TransformerEncoder.html

- **Relevant TensorRT Documentation**:

**Comment by Christina-Young-NVIDIA** [comment]
This issue needs to be subdivided into multiple tasks (too large for single sprint).

**Comment by apbose** [comment]
FX implementation update:

**Comment by apbose** [comment]
Added issues #1790, #1724, #1791, #1792, #1793, #1794

**Comment by apbose** [comment]
#1790 - No prior implementation in acc, will take more time
#1724 - Trivial operation since implementation in acc exists
#1791- Trivial operation since implementation in acc exists
#1792- Implementation exists, but tests might take time
#1793- No prior implementation in acc, will take more time
#1794 - Trivial operation since implementation in acc exists

**Comment by apbose** [comment]
Main operations left-
torch.ops.aten.sqrt
